support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -228,7 +219,7 @@ function buildRegularPaymentRequest(
 		csrUsername,
 		salesforceCaseId,
 		debugInfo: actionHistory,
-		threeTierCreateSupporterPlusSubscription: inThreeTier,
+		threeTierCreateSupporterPlusSubscription: false,
```

Comment:
Should we not remove this completely?

Reply:
Looks like it's actually optional [in the backend](https://github.com/guardian/support-frontend/blob/d678f7c635f346c97be132e986198ac20924c3fb/support-frontend/app/services/stepfunctions/SupportWorkersClient.scala#L66)? So we can safely remove it